src/diff/changes/apiChanges.test.ts

```diff
@@ -10,18 +10,26 @@ import { RuleCategory } from "../ruleSet";
 import { CategorizedChange } from "./categorizedChange";
 import { ApiChanges } from "./apiChanges";
 
-function buildApiChanges(): ApiChanges {
-  const titleChange = new CategorizedChange(
-    "r1",
-    "title-changed",
-    RuleCategory.BREAKING,
-    ["old-title", "new-title"]
-  );
-  const nodeChanges = new NodeChanges("test-id-1", ["type:title"]);
-  nodeChanges.categorizedChanges = [titleChange];
+function buildCategorizedChange(
+  c = RuleCategory.BREAKING,
+  id = 0
+): CategorizedChange {
+  const rule = `${c} Rule ${id}`;
+  return new CategorizedChange(rule, "title-changed", c, ["old", "new"]);
+}
+
+function buildNodeChanges(
+  categories = [RuleCategory.BREAKING],
+  id = 0
+): NodeChanges {
+  const nodeChanges = new NodeChanges(`test-id-${id}`, ["type:title"]);
+  nodeChanges.categorizedChanges = categories.map(buildCategorizedChange);
+  return nodeChanges;
+}
 
+function buildApiChanges(categories = [[RuleCategory.BREAKING]]): ApiChanges {
```

Comment:
I find it strange that we default these functions to having changes with breaking changes.  But just a nit.

Reply:
The original function always used a breaking change. I wanted to be able to specify a different type of change without changing the original function calls.